streaming/text/convert/README.md

```diff
@@ -1,17 +1,73 @@
 # Dataset preparation
 
-To use Streaming Dataset, We first convert the dataset from its native format to MosaicML's streaming dataset format (a collection of binary `.mds` files). Once in `.mds` format, we can store the dataset in a central location (filesystem, S3, GCS, etc.) and stream the data to any compute cluster, with any number of devices, and any number of CPU workers, and it all just works.
+To use Streaming Dataset we must first convert the dataset from its native format to MosaicML's Streaming Dataset format called Mosaic Data Shard (MDS). Once in MDS format, we can access the from the local file system ( disk network attached storage, etc.) or object store (GCS, OCS, S3, etc.).  From object store, data can be streamed to train deep learning models and it all just works efficiently.
```

Comment:
i suppose Mosaic Dataset Shard is clearer?

spurious space "( "

Reply:
Addressed in https://github.com/mosaicml/streaming/pull/183/commits/c022ee4c491d76088cbd44f4c6f39f865a46ad9c.